tests/OSPSuite.Core.Tests/Domain/InitialConditionsCreatorSpecs.cs

```diff
@@ -92,6 +92,37 @@ public void the_formula_used_should_be_the_formula_supplied_to_the_creator()
       }
    }
 
+   internal class When_creating_initial_conditions_in_a_container_with_parent_path_at_root : concern_for_InitialConditionsCreator
+   {
+      private MoleculeBuilder _molecule;
+      private Container _container;
+      private InitialCondition _initialCondition;
+
+      protected override void Context()
+      {
+         base.Context();
+         _molecule = new MoleculeBuilder().WithName("moleculeName").WithDimension(Constants.Dimension.NO_DIMENSION);
+         _container = new Container
+         {
+            ContainerType = ContainerType.Organ,
+            Mode = ContainerMode.Physical,
+            Name = "topContainer",
+            ParentPath = new ObjectPath("ItGoesSomewhere")
+         };
+      }
+
+      protected override void Because()
+      {
+         _initialCondition = sut.CreateInitialCondition(_container, _molecule);
+      }
+
+      [Observation]
+      public void the_initial_condition_should_have_root_container_parent_path()
+      {
+         _initialCondition.Path.PathAsString.ShouldBeEqualTo("ItGoesSomewhere|topContainer|moleculeName");
```

Comment:
well it won't crash because you are comparing the path as string but this is still wrong :)

Reply:
![image](https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/assets/261477/0627bfa5-670a-4765-acf1-ba22d2ddf6bd)

Oof!

Reply:
Ah ah told you